src/GregsStack.InputSimulatorStandard/GregsStack.InputSimulatorStandard.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>net40;netstandard2.0</TargetFrameworks>
+    <TargetFrameworks>net5.0;net6.0;netcoreapp3.1</TargetFrameworks>
```

Comment:
Please revert changes to `TargetFrameworks` as it will break compatibility with older applications.